searx/engines/hex.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""hex.pm"""
+
+from urllib.parse import urlencode
+from dateutil import parser
+
+
+about = {
+    "website": "https://hex.pm/",
+    "wikidata_id": None,
+    "official_api_documentation": "https://github.com/hexpm/hexpm/blob/main/lib/hexpm_web/controllers/api/package_controller.ex",
```

Comment:
```suggestion
    "official_api_documentation": "https://github.com/hexpm/hexpm/blob/main/lib/hexpm_web/controllers/api/package_controller.ex", # pylint: disable=line-too-long
```
fixes the CI / linter